wordpress-mobile/WordPress-Android
# Issue/13326 scan fixing in builder

Parent #13326

This PR moves threat fixing state item building logic to the builder class. It fixes the issue when progress bar was not displayed when an ignored threat was fixed.

It also 
- updates header, description
- removes labels from the progress bar 
- adds loading spinner on threat list items when threats are fixing
to have a bit of similarity with the iOS app.

https://user-images.githubusercontent.com/1405144/106699637-4303e680-6609-11eb-8baf-a32cc766b85f.mov

**To test**

Prerequisite:

- A threat is added to the testing site. In case you run out of threats, re-install bad calendar plugin and re-run scan.
- Scan feature flag is enabled

1. Open Scan screen
2. Start scan
3. Click on "Fix threats" button
4. Notice that 
    - app displays "Fixing Threats" state and progress indicator is displayed on the threat items
    - on pressing back and returning to the screen while threats are fixing, "Fixing Threats" state is re-shown
    - as threats get fixed, they are removed from the list
    - once fixing threats list is empty, scan state is re-fetched and displayed

** Also test the flow when an ignored threat is fixed with steps from PR #13924.

**Notes**
- Ignore styling

PR submission checklist:
